src/cmdlib.sh

```diff
@@ -288,6 +288,36 @@ commit_overlay() {
         --skip-list <(echo /statoverride)
 }
 
+create_content_manifest(){
+        # Convert the yaml file of content_sets to json format
+    mkdir -p "${workdir}"/tmp/buildinfo
+    yaml2json /srv/redhat-coreos/content_sets.yaml "${workdir}"/tmp/buildinfo/content_manifest.json
+    REPOS=$(python3 -c "
+import os, json;
+f = open('$workdir/tmp/buildinfo/content_manifest.json', 'r', encoding='utf-8');
+data = json.load(f);
+f.close();
+repos=[];
+for i in data['repo_mapping']:
+    for j in data['repo_mapping'][i]:
+        if(j=='name'):repos.append(data['repo_mapping'][i][j])
```

Comment:
This looks like it's outputting every repo in the file, but instead I think what we want is to parse the flattened manifest and only output the mapped repos which are used.